sympy/integrals/transforms.py

```diff
@@ -1169,6 +1734,59 @@ def _collapse_extra(self, extra):
                 'Laplace', None, 'No combined convergence.')
         return plane, cond
 
+    def doit(self, **hints):
+        """
+        Try to evaluate the Laplace transform in closed form.
+        """
+
+        needeval = hints.pop('needeval', False)
+        fn = self.function
+        t_ = self.function_variable
+        s_ = self.transform_variable
+
+        if not fn.is_Add:
+            fn = expand_mul(fn)
+
+        if not fn.is_Add:
+            try:
+                return self._compute_transform(fn, t_, s_, **hints)
+            except IntegralTransformError:
+                pass
+        else:
+            hints['needeval'] = needeval
+            res = [self.__class__(*([x] + list(self.args[1:]))).doit(**hints)
+                   for x in fn.args]
+            extra = []
+            ress = []
+            for x in res:
+                if not isinstance(x, tuple):
+                    x = [x]
```

Comment:
If `x` is not a tuple then turn it into a list?

Reply:
I don't know. This is a copy-paste from the present `doit` definition of the class `IntegralTransform`, I did not change that part.

Reply:
If https://github.com/sympy/sympy/pull/22376#discussion_r761809377 is solved, this one is resolved too, so I resolve it right away :)